src/main/kotlin/org/jboss/tools/intellij/kubernetes/tree/OpenShiftStructure.kt

```diff
@@ -83,4 +94,11 @@ class OpenShiftStructure(model: IResourceModel): AbstractTreeStructureContributi
             model
     )
 
+    private class ImageStreamDescriptor(element: ImageStream, parent: NodeDescriptor<*>?, model: IResourceModel) : TreeStructure.Descriptor<ImageStream>(
+            element,
+            parent,
+            { element.metadata.name },
+            IconLoader.getIcon("/icons/project.png"),
```

Comment:
Not sure why this is the same as projects but noticed the VSCode one has no icons for ImageStreams

Reply:
> Not sure why this is the same as projects but noticed the VSCode one has no icons for ImageStreams

I can remove the icons, there's no problem to that. I think that we should have good icons for all additional resources at some point. I just thought that I'd not care about icons for and take care of them altogether once we settled for good icons.